src/Indigo/Contracts/HomebaseLite/Impl/Vote.hs

```diff
@@ -21,8 +21,8 @@ propose :: HasSideEffects => HomebaseLiteEntrypoint ("proposal_uri" :! URI, "cho
 propose par = do
   description [itu|
     Called by a `participant` to submit a new proposal.
-    Takes as input an IPFS URI and a list of strings for the `n`  available choices.
-    A new proposal for the given IPFS will be initialized (with a `submitted` status).
+    Takes an IPFS URI and a list of strings for the available choices as input.
+    A new proposal for the given IPFS URI will be initialized (with a `submitted` status).
```

Comment:
I think we should remove the status here (there's no context for it) and instead mention the configuration options:
```suggestion
    A new proposal for the given IPFS URI will be initialized using the current
    **configuration options** (see `configure`).
```